### ISSUE TITLE
[$10][MSFT-31] [Challenge Feed] User can't focus on the Design. Development or Data Science toggles using tab

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2753]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges and Login using valid credentials
2. Press Tab key till Design. Development or Data Science is selected

**Expected Result**
User should be able to focus on the Design. Development or Data Science toggles using tab Key

**Actual Result**
User can't focus on the Design. Development or Data Science toggles using tab Key

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F16

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096079 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096079 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by r0hit-gupta** [comment]
Please assign me

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096079 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097521 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097521 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by r0hit-gupta** [comment]
@veshu @codeMinter issue is duplicate of https://github.com/topcoder-platform/community-app/issues/2746. PR was https://github.com/topcoder-platform/community-app/pull/2939 was closed due to this. Please reopen and review. Thanks.

**Comment by codeMinter** [comment]
@r0hit-gupta Yes they are duplicates, so we should have PR for one only. The other just put comment it is duplicate and QA will close it.

**Comment by r0hit-gupta** [comment]
Duplicate of #2746

**Comment by crazyk07** [comment]
Issue is duplicate and now solved in Prod

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
